server/app/services/auth-service.js

```diff
@@ -32,16 +32,35 @@ async function isTokenRevoked(tokenId) {
  * Create a new JWT including permissions.
  *
  * @param user {ConnectionInformation} User
+ * @param rememberMe {ConnectionInformation} User
  * @returns {Promise<String>} Return a JWT.
  */
-async function createJWT(user) {
+async function createJWT(user, rememberMe) {
   // Sign with default (HMAC SHA256)
   const id = uuidv4();
 
   await user.createJwt({
     id,
   });
 
+  logger.info(`Creating a new JWT ${user.id}`);
+
+  const exp = rememberMe ? Math.floor(Date.now() / 1000) + (60 * 60 * 730) // One mounth
```

Comment:
Like said on #250, I was thinking about letting the client set his duration in term of days. Then just limit the duration with Joi.

Instead of taking a remember me field which does not mean much on a Rest API.